utils/mock_qpu/ionq/__init__.py

```diff
@@ -7,26 +7,29 @@
 # ============================================================================ #
 
 import cudaq
-from fastapi import FastAPI, Request, HTTPException, Header
-from typing import Optional, Union
-import uvicorn, uuid, json, base64, ctypes
+from fastapi import FastAPI, HTTPException, Header
```

Comment:
Doesn't need changing in this script but [relevant issue](https://github.com/NVIDIA/cuda-quantum/issues/408)